gubled/gubled_test.go

```diff
@@ -140,7 +140,13 @@ func TestGcmOnlyStartedIfEnabled(t *testing.T) {
 	defer mockCtrl.Finish()
 
 	routerMock := NewMockPubSubSource(mockCtrl)
+
+	amMock := NewMockAccessManager(mockCtrl)
+	msMock := NewMockMessageStore(mockCtrl)
+
 	routerMock.EXPECT().KVStore().Return(store.NewMemoryKVStore(), nil)
+	routerMock.EXPECT().AccessManager().Return(amMock, nil).MaxTimes(2)
+	routerMock.EXPECT().MessageStore().Return(msMock, nil).MaxTimes(2)
```

Comment:
Would be the same with the generated mock. 
I would never leave the mock code in the production code, so I see two possible solutions:
1. have the mock redundant in the _test.go code of each package, where we need it
2. Create a separate test_util package

In this small case I would prefer the redundancy variant.